src/pages/notice_list.tsx

```diff
@@ -0,0 +1,11 @@
+import { Layout } from 'components/Layout';
+import type { NextPage } from 'next';
+import Link from 'next/link';
+
+const Home: NextPage = () => (
```

Comment:
```suggestion
const Notices: NextPage = () => (
```

파일 이름도 `notice_list` 가 아니라 보통 url 에서는 `notices` 라고 씀
따라서 컴포넌트 이름도 `Notices`